### DISCUSSION TITLE
Securityonion suricata is not sending logs to -rw-r--r-- 1 suricata suricata    0 Nov 14 10:25 eve-2022-11-14-10:25.json

### DISCUSSION THREAD

**Discussion by kunalhiremath** [Security-Onion-Solutions/securityonion#9141] [Unsupported Versions]
I am a beginner and I was trying out distributed deployment of securityonion, everything is working fine but suricata is not logging the logs (-rw-r--r-- 1 suricata suricata    0 Nov 14 10:25 eve-2022-11-14-10:25.json ) and also no logs are getting populated on kibana dashboard.

Can anyone help me with this.
Thank you.

**Comment by dougburks** [comment]
Start with the Suricata troubleshooting section of our documentation:
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts
